### PR TITLE
Fix post downloads summary command

### LINKED ISSUE #2215: Engagement bot summary doesn't seem to be working
### Context

https://github.com/AlexsLemonade/refinebio/issues/2140

### Problem or idea

Tested today and reported no downloads in the last 7 days.

> **EngagementBot**
> There were no downloads in the last 7 days.

However there were a few that should have been counted:

```
data_refinery=> select id, created_at from data_refinery_common_dataset where is_processed='t' and email_address is not NULL order by created_at desc limit 10;
                  id                  |          created_at           
--------------------------------------+-------------------------------
 381e86b9-aea2-4d9a-8a61-338b111fe54d | 2020-03-29 13:06:28.954778+00
 eb726da7-8006-46c4-95ba-8a3bcecbd6f1 | 2020-03-26 02:26:35.196333+00
 099e7b58-dce6-4d5c-a4b6-f4d049dad866 | 2020-03-25 16:19:14.656207+00
 c4c086ef-6963-4fd8-b4e2-a7cf09c9eb68 | 2020-03-25 16:18:03.158765+00
 0716857b-211f-484c-9e2b-3e447dd5cdaf | 2020-03-25 15:34:20.155269+00
 2e7c1e1c-138b-43b1-9463-0113c64e7edb | 2020-03-25 15:19:23.577922+00
 4a2ac1ef-4e9e-4945-bdf8-00d595f99c4c | 2020-03-25 15:18:21.49652+00
 bbb30f4e-7b5f-4595-a47d-25f8d72e4a7c | 2020-03-25 15:03:55.391917+00
 eff9cbe4-c4f5-43a6-b004-6cd36f10ea15 | 2020-03-24 22:41:51.185922+00
```

### Solution or next step

Investigate why these downloads are not being reported.

### PR DESCRIPTION
**Pull request by arielsvn**
## Issue Number

close #2215 

## Purpose/Implementation Notes

I made an error when calculating the `start_date` from which to get the stats. This fixes it and also includes the downloader country in the summary.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Functional tests

Tested in prod, and posted a message to slack.

## Checklist

- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

File: api/data_refinery_api/management/commands/post_downloads_summary.py [hunk 1]
```diff
@@ -122,7 +121,7 @@ def should_display_email(email: str) -> bool:
 
 def get_ip_location(remote_ip):
     try:
-        city = requests.get("https://ipapi.co/" + remote_ip + "/json/", timeout=10).json()["city"]
+        data = requests.get("https://ipapi.co/" + remote_ip + "/json/", timeout=10).json()
```
Comment (kurtwheeler) [inline api/data_refinery_api/management/commands/post_downloads_summary.py]: It just occurred to me that we're doing this at summary time only. This means we don't have the address stored anywhere. IP addresses change over time, which means that the only persistent store of locations we've had downloads from is the physcial maps in our office?

Comment (arielsvn) [reply api/data_refinery_api/management/commands/post_downloads_summary.py]: hmm does the location of an IP address change?

Comment (kurtwheeler) [reply api/data_refinery_api/management/commands/post_downloads_summary.py]: Yes, apparently by about 10% by year: https://stackoverflow.com/questions/23493812/are-geolocation-ip-ranges-changing-over-the-years

This means that if we ever wanna look back at our downloads over the years our location data won't be useful.

Comment (arielsvn) [reply api/data_refinery_api/management/commands/post_downloads_summary.py]: ah, good find! In that case, let me create a different issue to save these locations when we calculate them.

Comment (arielsvn) [reply api/data_refinery_api/management/commands/post_downloads_summary.py]: created https://github.com/AlexsLemonade/refinebio/issues/2217

File: api/data_refinery_api/management/commands/post_downloads_summary.py [hunk 2]
```diff
@@ -105,14 +105,13 @@ def should_display_email(email: str) -> bool:
...
         return False
-
     return not (
         email.startswith("cansav09")
         or email.startswith("arielsvn")
         or email.startswith("jaclyn.n.taroni")
         or email.startswith("kurt.wheeler")
         or email.startswith("greenescientist")
-        or "@alexslemonade.org" not in email
+        or "@alexslemonade.org" in email
```
Comment (davidsmejia) [inline api/data_refinery_api/management/commands/post_downloads_summary.py:114]: Does this mean we are now including `alexslemonade` emails in the summaries? I was using mine for testing purposes, i must have created 100 datasets this sprint.

Comment (arielsvn) [reply api/data_refinery_api/management/commands/post_downloads_summary.py:114]: No, in this case, there was a double negation. Note that there's a `not` also at the beginning of the condition. Should we use a variable to make this easier to see?

Comment (davidsmejia) [reply api/data_refinery_api/management/commands/post_downloads_summary.py:114]: i would just remove it and add email.endsWith("@alexslemonade.org") after the begins with rules

Comment (arielsvn) [reply api/data_refinery_api/management/commands/post_downloads_summary.py:114]: Sure!

```suggestion
        or email.endswith("@alexslemonade.org")
```

File: api/data_refinery_api/management/commands/post_downloads_summary.py [hunk 3]
```diff
@@ -105,14 +105,13 @@ def should_display_email(email: str) -> bool:
...
     return not (
         email.startswith("cansav09")
         or email.startswith("arielsvn")
         or email.startswith("jaclyn.n.taroni")
         or email.startswith("kurt.wheeler")
         or email.startswith("greenescientist")
-        or "@alexslemonade.org" not in email
+        or email.endswith("@alexslemonade.org")
         or email.startswith("miserlou")
         or email.startswith("d.prasad")
```
Comment (davidsmejia) [inline api/data_refinery_api/management/commands/post_downloads_summary.py:116]:
```suggestion
        or email.startswith("miserlou")
        or email.startswith("d.prasad")
        or email.endswith("@alexslemonade.org")
```

Comment (arielsvn) [reply api/data_refinery_api/management/commands/post_downloads_summary.py:116]: ah consistency!

**Pull request merged**